<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IzzyTime</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 158, 100);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btCurDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(198, 158, 100);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="twYear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 158, 100);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
selection-background-color: rgb(143, 107, 52);
color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twMonth">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>31</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 158, 100);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
selection-background-color: rgb(143, 107, 52);
color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twDay">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 158, 100);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
selection-background-color: rgb(143, 107, 52);
color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twTaskField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>301</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twHangedTaskField">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>301</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDay">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>31</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(198, 158, 100);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btWeek">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>31</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(198, 158, 100);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btCalendar">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>360</y>
      <width>31</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(198, 158, 100);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSelDate">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>2</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Selected Date</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
